frontend/public/signup.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background: url('https://rabblerouserteam.files.wordpress.com/2016/08/rabble-rouser-header.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  background-color: #EA3B49;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding-left: 50px;
+}
+
+header img {
+  content:url('https://i.imgur.com/eecsJFb.png');
```

Comment:
I hadn't seen the content:url approach before. 
The url could be provided in the component which is maybe not as consistent.

Otherwise ditching the img tag and a combo of background-image, background-size, background-color and height on the header should achieve the same effect.